vortex-array/src/arrays/varbinview/vtable/operator.rs

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use vortex_buffer::Buffer;
+use vortex_compute::filter::Filter;
+use vortex_dtype::DType;
+use vortex_error::VortexResult;
+use vortex_vector::Vector;
+use vortex_vector::binaryview::{BinaryVector, BinaryView, BinaryViewTypeUpcast, StringVector};
+
+use crate::ArrayRef;
+use crate::arrays::{VarBinViewArray, VarBinViewVTable};
+use crate::execution::{BatchKernelRef, BindCtx, kernel};
+use crate::vtable::{OperatorVTable, ValidityHelper};
+
+impl OperatorVTable<VarBinViewVTable> for VarBinViewVTable {
+    fn bind(
+        array: &VarBinViewArray,
+        selection: Option<&ArrayRef>,
+        ctx: &mut dyn BindCtx,
+    ) -> VortexResult<BatchKernelRef> {
+        let mask = ctx.bind_selection(array.len(), selection)?;
+        let validity = ctx.bind_validity(array.validity(), array.len(), selection)?;
+        let dtype = array.dtype().clone();
+
+        let views = array.views().clone();
+        let buffers = array.buffers().clone();
+
+        Ok(kernel(move || {
+            let selection = mask.execute()?;
+            let validity = validity.execute()?;
+
+            // We only filter the views buffer
+            let views = views.filter(&selection);
+            let views = Buffer::<BinaryView>::from_byte_buffer(views.into_byte_buffer());
```

Comment:
What does this line do?

Reply:
transmutes from a `Buffer<vortex_array::BinaryView>` to `Buffer<vortex_vector::BinaryView>`